_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,205 @@
+---
+name: iframe has a unique accessible name
+
+description: |
+ The accessible name for each iframe is unique
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to `iframe` elements that are [exposed to assistive technologies](#exposed-to-assistive-technologies) and have a [non-empty](#non-empty) [accessible name](#accessible-name).
+
+### Expectation
+
+Each target element has an [accessible name](#accessible-name) that is unique across all other accessible names for `iframe` elements within the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees), where the `src` attributes are different.
```

Comment:
This shouldn't be "Unique across all ... `iframe` elements". That effectively says to ignore "exposed to assistive technologies". It should probably be something like:

> Each target element has an [accessible name](#accessible-name) that is unique for all applicable iframes within the same ...